MvvmCross/Platforms/Uap/Presenters/MvxWindowsViewPresenter.cs

```diff
@@ -240,5 +251,36 @@ protected virtual Task<bool> ShowPage(Type viewType, MvxBasePresentationAttribut
                 return Task.FromResult(false);
             }
         }
+
+        protected virtual async Task<bool> ShowModal(Type viewType, MvxModalViewPresentationAttribute attribute, MvxViewModelInstanceRequest request)
+        {
+            try
+            {
+                if (Activator.CreateInstance(viewType) is MvxWindowsContentDialog modalView)
```

Comment:
Isn't there a view creator for Windows? @nickrandolph

Reply:
ContentDialog is created with the new command. But if you know another way to do it, propose it.

Reply:
The ContentDialog not is a page.  I can create a method to create the ContentDialog. I can't think of anything else.